### PR TITLE
Fix static IPs config option and use a new name

### PR DESCRIPTION
**Pull request by osherdp**
Fixed ``IS_STATIC_IP`` configuration option, and changed its name to just ``STATIC_IPS``.

Seems like ``static_network_config`` option not delivered to ``prepare_for_installation`` so that we always just configured dynamic networking.
Also, we only gather IPs via DHCP server, although libvirt is capable to get it via ARP (which is useful for static IPs).

/cc @eliorerz @YuviGold 
/hold

### CODE REVIEW

File: discovery-infra/test_infra/utils/global_variables/env_variables_utils.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+from abc import ABC
```
Comment (YuviGold) [inline discovery-infra/test_infra/utils/global_variables/env_variables_utils.py:1]: should be removed

**Review by YuviGold — APPROVED** [review]

**Comment by osherdp** [comment]
It's still not fixing it all, need to keep working on the fix
/retitle [WIP] Fix static IPs config option and use a new name

**Comment by osherdp** [comment]
/retitle Fix static IPs config option and use a new name

**Review by YuviGold — APPROVED** [review]

**Pull request merged**